libphal/libphal.H

```diff
@@ -151,6 +151,12 @@ void getDump(struct pdbg_target *proc, const uint8_t type, const uint8_t clock,
  */
 void threadStopProc(struct pdbg_target *proc);
 
+/**
+ * @brief Handle SBE error
+ * @param[in] i_procTarget Non NULL proc Target handle.
+ * @return Exception thrown on failure
+ */
+void sbeErrorHandler(struct pdbg_target *proc);
```

Comment:
yeah good point, it should not be an interface, as per our discussion it should be called from captureFFDC. Others will just call captureFFDC. that is the direction

Reply:
IMO, we are losing the API useability.

OR

Please update captureFFDC interface documentation and also, we need to check all caller's current use cases because you are changing the one of the library interface responsebility.